app/src/services/realitio.ts

```diff
@@ -45,11 +47,14 @@ class RealitioService {
    */
   askQuestion = async (
     question: string,
+    category: string,
     openingDateMoment: Moment,
     value = '0',
   ): Promise<string> => {
     const openingTimestamp = openingDateMoment.unix()
-    const args = [0, question, this.arbitratorAddress, '86400', openingTimestamp, 0]
+    const questionText = RealitioQuestionLib.encodeText('bool', question, [1, 0], category)
```

Comment:
From what I see in the tests of the library (there doesn't seem to be any documentation), `[1, 0]` can (should?) be replaced with `null`.

Reply:
Added null for outcomes parameter